plugins/isColor.js

```diff
@@ -0,0 +1,3 @@
+export default function isColor(strColor){
```

Comment:
i'm not sure if `plugins` is the right folder name here
what about just using `utils` or something similar?

Reply:
utils sounds good!